eventsProject/singleEventPage.php

```diff
@@ -7,32 +7,62 @@
 	wrongPage();
 }
 
-$event=getOneEvent($_REQUEST['eventID']); //should be $_REQUEST
+$eventID=($_REQUEST['eventID']);
+$event=getOneEvent($eventID);
 $categories=getCatsForThisEvent($_REQUEST['eventID']);
 $usersGoing=getUsersGoingToThisEvent($_REQUEST['eventID']);
 
 $creator=getWhoCreatedEvent($_REQUEST['eventID']);
 
 echoHeader($event['name'], null);
 
+$myEventVotes=$event['votes'];
 
-// <div class='centeredPictureText'>
-// <div class='row'>
-// 	<div class='left'>
-// 		$event[votes]
-// 	</div>
-// 	<div class='right'>
-// 		$event[name]
-// 	</div>
-// </div>
+?>
+<script src="/include/jquery.js"></script>
```

Comment:
You can just include this from the `echoHeader` function. You might as well just always have jquery available.